apps/nextra/pages/en/build/create-aptos-dapp.mdx

```diff
@@ -42,6 +42,16 @@ cd your/workspace
     pnpx create-aptos-dapp@latest
     ```
   </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash filename="Terminal"
+    yarn create aptos-dapp
```

Comment:
```suggestion
    yarn create-aptos-dapp
```

Do we also want `@latest` on this?

Reply:
> Do we also want `@latest` on this?

This is the correct approach: when creating templates with Yarn, using absolute paths can cause failures.

Yarn will automatically use the latest template when creating a project.

Reply:
This still needs to have the dash right?